WalletWasabi/Extensions/VerboseBlockInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace NBitcoin.RPC
+{
+	public class VerboseBlockInfo
+	{
+		public DateTimeOffset BlockTime { get; set; }
+		public uint256 Hash { get; set; }
+		public uint256 PrevBlockHash { get; set; }
+		public ulong Confirmations { get; set; }
+		public ulong Height { get; set; }
+		public List<VerboseTransactionInfo> Transactions { get; set; } = new List<VerboseTransactionInfo>();
```

Comment:
These properties should be `get` only with proper constructors and the list should be an `IEnumerable`.